libafl_targets/src/cmps/observers/aflpp.rs

```diff
@@ -285,7 +267,7 @@ impl<'a> CmpObserverMetadata<'a, AFLppCmpLogMap> for AFLppCmpValuesMetadata {
 
                 let cmpmap_idx = i;
                 let mut cmp_values = Vec::new();
-                if cmp_observer_data {
+                if self.original {
```

Comment:
Trusting these changes, I'm not sure what is expected here.

Reply:
no this part is not complete yet. (this code won't compile)
since it is in libafl-targets. i'll do later